packages/editor/src/components/url-popover/style.scss

```diff
@@ -9,19 +9,53 @@
 		flex-grow: 1;
 	}
 
+	// Mimic toolbar component styles for the icons in this popover
+	.components-icon-button {
+		padding: 3px;
+
+		> svg {
+			padding: 5px;
+			border-radius: $radius-round-rectangle;
+			height: 30px;
+			width: 30px;
+		}
+
+		&:not(:disabled):not([aria-disabled="true"]):not(.is-default):hover {
+			box-shadow: none;
+
+			> svg {
+				@include formatting-button-style__hover;
+			}
+		}
+
+		&:not(:disabled):focus {
+			box-shadow: none;
+
+			> svg {
+				@include formatting-button-style__focus;
+			}
+		}
+	}
+
 	&__settings-toggle {
```

Comment:
I know you didn't write this initially, but while you're here do you think we should change this rule to be less obtuse? Although there isn't any written rule saying this much, in general I believe the SCSS standard pattern that is emerging is to use write `.editor-url-popover__settings-toggle` rather than `.editor-url-popover { &__settings-toggle { ... } }`.

Hmm actually that might be a big rewrite of this CSS file. Okay to ignore if you feel this is too big of a task.

Reply:
Agreed! This was really confusing to parse. I'll take care of this in a separate PR, so we can get this change in first. 👍